GitCommands/Remote/GitRemote.cs

```diff
@@ -1,4 +1,5 @@
 using System.Collections.Generic;
+using GitCommands.Config;
```

Comment:
Looks strange, but was needed to fix type references in XML docs.

Reply:
This is part of why SA0001 is so important to me. 😁